rasr/feature_scorer.py

```diff
@@ -83,7 +84,11 @@ def __init__(
         else:
             self.config.normalize_mixture_weights = False
         self.config.pymod_name = "returnn.SprintInterface"
-        self.config.pymod_path = os.path.join(tk.gs.RETURNN_ROOT, "..")
+        self.config.pymod_path = (
+            returnn_root
+            if returnn_root is not None
+            else os.path.join(gs.RETURNN_ROOT, "..")
```

Comment:
A) can we make it a tk.Path object also in the else branch?
B) would we also need to add the .. if a returnn_root was given by the user?
C) do you need to add something below to handle tk.Path vs. str?

Reply:
A) yes
B) the RETURNN dir structure changed at some point.. 
```
returnn
  - code
```
now
```
returnn
   - returnn
     - code
```
so I think for the current setup, which I tested you do not need it.. I did not test the old setup..

C) could you clarify where exactly??

D) I added typing but this somehow broke some tests.. this would need investigation

Reply:
B) Old setup means older than 2 year old RETURNN. 
So if people set this by hand, I am quite certain they use a new version.

Reply:
And what was the issue with the typing?

Reply:
C) sorry I was just lazy to check. Probably nothing needs to be done after second thought.

B) are we now implicitly assuming that gs.RETURNN_ROOT implies old folder structure and giving directly implies new folder structure?
Or would the new structure also work with the two dots as in gs.RETURNN_ROOT?

Reply:
> And what was the issue with the typing?

I added typing and some tests failed because import failed. still not sure why.. 

B) I think it should work but I am not sure.. I did not test

Reply:
Ok. concerning A) I still suggest
```suggestion
            else tk.Path(os.path.join(gs.RETURNN_ROOT, ".."))
```
or
```suggestion
            else tk.Path(gs.RETURNN_ROOT).join_right("..")
```